pages/album/[id].js

```diff
@@ -1,30 +1,56 @@
 import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
 import AlbumDetail from "../../components/AlbumDetail/AlbumDetail.js";
-import curtisAlbum from "../../public/images/curtis.jpeg";
 import AppHeader from "../../components/Header/Header.js";
 
 export default function AlbumDetailPage() {
   const router = useRouter();
-  const { id } = router.query;
+  const { id, coverImage } = router.query;
```

Comment:
thus you do not need the cover image here

```suggestion
  const { id } = router.query;
```